manpages/fsck.fat.8.in

```diff
@@ -103,125 +103,129 @@ The other is the legacy Atari variant used on Atari ST.
 .IP "" 4
-Previous versions of this program exceptionally treated "EA DATA . SF" and
-"WP ROOT . SF" as valid short names; using this option does not preserve that
-exception.
-.IP "\fB\-t\fR" 4
+Previous versions of this program exceptionally treated \fIEA DATA . SF\fP and
+\fIWP ROOT . SF\fP as valid short names; using this option does not preserve
+that exception.
+.IP "\fB\-t\fP" 4
 Mark unreadable clusters as bad.
-.IP "\fB\-u\fR \fIPATH\fR" 4
+.IP "\fB\-u\fP \fIPATH\fP" 4
 Try to undelete the specified file.
-\fBfsck.fat\fR tries to allocate a chain of contiguous unallocated clusters
+\fBfsck.fat\fP tries to allocate a chain of contiguous unallocated clusters
 beginning with the start cluster of the undeleted file.
 This option can be given more than once.
-.IP "\fB\-v\fR" 4
+.IP "\fB\-v\fP" 4
 Verbose mode.
 Generates slightly more output.
-.IP "\fB\-V\fR" 4
+.IP "\fB\-V\fP" 4
 Perform a verification pass.
 The filesystem check is repeated after the first run.
 The second pass should never report any fixable errors.
 It may take considerably longer than the first pass, because the first pass may
 have generated long list of modifications that have to be scanned for each disk
 read.
-.IP "\fB\-\-variant\fR \fITYPE\fR" 4
-Create a filesystem of variant \fITYPE\fR.
-Acceptable values are 'standard' and 'atari' (in any combination of upper/lower
-case).
+.IP "\fB\-\-variant\fP \fITYPE\fP" 4
+Create a filesystem of variant \fITYPE\fP.
+Acceptable values are \fIstandard\fP and \fIatari\fP (in any combination of
+upper/lower case).
 See above under DESCRIPTION for the differences.
-.IP "\fB\-w\fR" 4
+.IP "\fB\-w\fP" 4
 Write changes to disk immediately.
-.IP "\fB\-y\fR" 4
-Same as \fB\-a\fR (automatically repair filesystem) for compatibility with other
+.IP "\fB\-y\fP" 4
+Same as \fB\-a\fP (automatically repair filesystem) for compatibility with other
 fsck tools.
-.IP "\fB\-\-help\fR" 4
+.IP "\fB\-\-help\fP" 4
 Display help message describing usage and options then exit.
 .\" ----------------------------------------------------------------------------
 .SH "EXIT STATUS"
 .IP "0" 4
 No recoverable errors have been detected.
 .IP "1" 4
-Recoverable errors have been detected or \fBfsck.fat\fR has discovered an
+Recoverable errors have been detected or \fBfsck.fat\fP has discovered an
 internal inconsistency.
 .IP "2" 4
 Usage error.
-\fBfsck.fat\fR did not access the filesystem.
+\fBfsck.fat\fP did not access the filesystem.
 .\" ----------------------------------------------------------------------------
 .SH FILES
-.IP "fsck0000.rec, fsck0001.rec, ..." 4
-When recovering from a corrupted filesystem, \fBfsck.fat\fR dumps recovered data
-into files named 'fsckNNNN.rec' in the top level directory of the filesystem.
+.IP "\fIfsck0000.rec\fP, \fIfsck0001.rec\fP, ..." 4
+When recovering from a corrupted filesystem, \fBfsck.fat\fP dumps recovered data
+into files named \fIfsckNNNN.rec\fP in the top level directory of the filesystem.
```

Comment:
Conventionally, file names are set in italics, so:

    .SH FILES
    .TP
    .IR fsck0000.rec ", " fsck0001.rec ", " ...
    When ...